### PR TITLE
[Jenkins parser job] Skip auto retry if Jenkins already retries

### PR DESCRIPTION
**Pull request by aandvalenzuela**

### CODE REVIEW

File: jenkins/parser/jenkins-parser-job.py [hunk 1]
```diff
@@ -131,7 +131,24 @@ def get_new_running_builds(job_dir, last_processed_log):
     ]
 
 
-def trigger_retry_action(job_to_retry, build_to_retry, action):
+def trigger_retry_action(job_to_retry, build_to_retry, build_dir_path, action):
+    # Skip autoretry if Jenkins already retries
+    if grep(os.path.join(build_dir_path, "build.xml"), "<maxSchedule>", True):
+        print("... Jenkins already takes care of retrying. Skipping ...")
+        if grep(os.path.join(build_dir_path, "build.xml"), "<retryCount>", True): return
```
Comment (smuzaffar) [inline jenkins/parser/jenkins-parser-job.py:138]: @aandvalenzuela , is this check really needed?

File: jenkins/parser/jenkins-parser-job.py [hunk 2]
```diff
@@ -131,7 +131,24 @@ def get_new_running_builds(job_dir, last_processed_log):
     ]
 
 
-def trigger_retry_action(job_to_retry, build_to_retry, action):
+def trigger_retry_action(job_to_retry, build_to_retry, build_dir_path, action):
+    # Skip autoretry if Jenkins already retries
+    if grep(os.path.join(build_dir_path, "build.xml"), "<maxSchedule>", True):
+        print("... Jenkins already takes care of retrying. Skipping ...")
```
Comment (smuzaffar) [inline jenkins/parser/jenkins-parser-job.py:137]: why not just return from here?

Comment (smuzaffar) [reply jenkins/parser/jenkins-parser-job.py:137]: ah ok, I understand the logic now, forget my last comment

**Comment by smuzaffar** [comment]
looks good

**Comment by aandvalenzuela** [comment]
Thanks @smuzaffar!

**Pull request merged**